src/components/common/ColonyActions/helpers/getActionTitleValues.ts

```diff
@@ -199,9 +208,19 @@ const useGetActionTitleValues = ({
   );
   const keys = getMessageDescriptorKeys(actionType);
 
-  return generateMessageValues(updatedItem, keys, {
+  const messageValues = generateMessageValues(updatedItem, keys, {
     actionType,
   });
+
+  if (actionType.includes(ColonyActionType.MakeArbitraryTransaction)) {
+    const arbitraryMessageValues =
+      getFormatValuesArbitraryTransactions(actionData);
+    messageValues.arbitraryTransactionsLength =
+      arbitraryMessageValues.arbitraryTransactionsLength;
+    messageValues.arbitraryMethod = arbitraryMessageValues.arbitraryMethod;
+  }
+
+  return messageValues;
```

Comment:
Sorry I missed that the first time, I believe these should be added to `useMapColonyActionToExpectedFormat` hook

Reply:
you are right, thanks! fixed 👍